### PR TITLE
Supporting backward compatibillity

### PR DESCRIPTION
**Pull request by nbhoski**
supporting backward compatibility for newly  added artifacts.

### CODE REVIEW

File: src/main/java/com/mathworks/ci/RunMatlabTestsBuilder.java [hunk 1]
```diff
@@ -140,7 +149,42 @@ private void setEnv(EnvVars env) {
...
+            this.coberturaArtifact = new CoberturaArtifact("matlabTestArtifacts/cobertura.xml");
+        }
+        if (stmResultsChkBx) {
+            this.stmResultsArtifact =
+                    new StmResultsArtifact("matlabTestArtifacts/simulinktestresults.mldatx");
+        }
+        if (modelCoverageChkBx) {
+            this.pdfReportArtifact =
+                    new ModelCovArtifact("matlabTestArtifacts/coberturamodelcoverage.pdf");
+        }
```
Comment (mcafaro) [inline src/main/java/com/mathworks/ci/RunMatlabTestsBuilder.java:183]: Why is this creating a `pdfReportArtifact`?

Comment (nbhoski) [reply src/main/java/com/mathworks/ci/RunMatlabTestsBuilder.java]: Nice Catch :P  Updated it

Comment (mcafaro) [reply src/main/java/com/mathworks/ci/RunMatlabTestsBuilder.java]: Shouldn't it have `.xml` extension as well?

Comment (nbhoski) [reply src/main/java/com/mathworks/ci/RunMatlabTestsBuilder.java]: Yep :) done 1

Comment (mcafaro) [reply src/main/java/com/mathworks/ci/RunMatlabTestsBuilder.java]: Also, why is this a different value than the default in jelly? One is `coberturamodelcoverage.xml` and the other is `modelcoverage.xml`. Seems like these should be the same.

https://github.com/mathworks/jenkins-matlab-plugin/blob/a8f6ce39a9582e6dc887c7f69b9dc20220a254af/src/main/resources/com/mathworks/ci/RunMatlabTestsBuilder/config.jelly#L38-L42

Comment (nbhoski) [reply src/main/java/com/mathworks/ci/RunMatlabTestsBuilder.java]: This change is already merged in last PR #92

Comment (mcafaro) [reply src/main/java/com/mathworks/ci/RunMatlabTestsBuilder.java]: Which one is right?

Comment (nbhoski) [reply src/main/java/com/mathworks/ci/RunMatlabTestsBuilder.java]: The One which is merged is correct . That why i did not include that file in this PR   	
below is correct and already merged in the qualification branch 
  <f:textbox default="matlabTestArtifacts/coberturamodelcoverage.xml"/>

Comment (nbhoski) [reply src/main/java/com/mathworks/ci/RunMatlabTestsBuilder.java]: https://github.com/mathworks/jenkins-matlab-plugin/blob/6f5ac8c8a322a8b96ae1d1e1df65178e18c2ac3c/src/main/resources/com/mathworks/ci/RunMatlabTestsBuilder/config.jelly#L40

**Pull request merged**